pkg/configtx/consortiums.go

```diff
@@ -21,6 +21,22 @@ type Consortium struct {
 	Organizations []Organization
 }
 
+// AddConsortium adds a new consortium to a channel configuration.
+func (c *ConfigTx) AddConsortium(consortium Consortium) error {
+	consortiumsGroup := c.updated.ChannelGroup.Groups[ConsortiumsGroupKey]
+
+	consortiumsGroup.Groups[consortium.Name] = newConfigGroup()
+
+	for _, org := range consortium.Organizations {
```

Comment:
This logic implies that when adding a consortium the consumer will also include the orgs for the consortium. I'm fine with that and even if no orgs are specified they can be added later.